Source/WebCore/Modules/mediasource/MediaSource.cpp

```diff
@@ -185,73 +186,98 @@ const PlatformTimeRanges& MediaSource::buffered() const
     return m_buffered;
 }
 
-void MediaSource::seekToTime(const MediaTime& time)
+void MediaSource::seekToTarget(const SeekTarget& target, CompletionHandler<void(const MediaTime&)>&& completionHandler)
 {
     if (isClosed())
         return;
```

Comment:
Should we `seekCompletedHandler(MediaTime::invalidTime())` if closed?

Reply:
This is done when the MediaSource readyState changes to closed.

Reply:
As discussed, added the `AsyncReplyError<WTF::MediaTime>` template to handle this